docs/product/performance/requests.mdx

```diff
@@ -21,24 +21,15 @@ On the [**Requests** page](#requests-page), you get an overview of the domains t
   ></iframe>
 </div>
 
-## Prerequisites
+## Prerequisites and Limitations
 
 <Note>
 Network requests using non-HTTP protocols (FTP, WebSocket, etc.) are not supported at this time.
 </Note>
 
 Availability of HTTP request monitoring depends on the SDK your application uses. In most cases, Sentry's SDKs automatically enable HTTP request tracking. You can check your SDK's <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation documentation</PlatformLink> to see if it includes tracking HTTP requests.
 
-### Span Eligibility
-
-If the SDK you are using doesn't automatically instrument HTTP requests, you can use <PlatformLink to="/performance/instrumentation/custom-instrumentation/">custom instrumentation</PlatformLink>. For best results, follow these guidelines for your custom spans:
-
-- The `op` field is set to `"http.client"`.
-- The `description` field contains the HTTP method and the full URL of the request (e.g., `"GET http://my.service.io/some-data"`).
-- If the request is to a relative URL (e.g., `"GET /data.json"`), the `server.address` span data value should be set to the server address (e.g., `"my.service.io"`).
-- The `http.response.status_code` contains the HTTP response status code (e.g., `"200"`).
-
-See the [Span Data Conventions](https://develop.sentry.dev/sdk/performance/span-data-conventions/#http) page for more information.
+If the SDK you are using doesn't automatically instrument HTTP requests, you can use <PlatformLink to="/performance/instrumentation/requests-module/">custom instrumentation</PlatformLink> instructions to instrument your application. If your platform doesn't have dedicated custom instrumentation instructions, try adapting the code from another platform, since all the concepts are the same.
```

Comment:
```suggestion
If the SDK you're using doesn't automatically instrument HTTP requests, you can instrument your application by following the <PlatformLink to="/performance/instrumentation/requests-module/">custom instrumentation</PlatformLink> instructions. Because all the concepts are the same, if your platform doesn't have dedicated custom instrumentation instructions, you can try adapting the code from another platform.
```